impl-endpoints/cloudflare-go/run_endpoint.sh

```diff
@@ -8,7 +8,7 @@ if [ "$ROLE" = "client" ]; then
     echo "Running Cloudflare-Go client."
     echo "Client params: $SERVER_PARAMS"
     echo "Test case: $TESTCASE"
-    /cf-go/bin/go run /client.go
+    /cf-go/bin/go run /runner.go -role=client -test=${TESTCASE}
```

Comment:
Hmmm, perhaps this file can be removed and we can just pass arguments to the runner through Dockerfile `ARG`s. However, setting up Wireshark and ns-3 might require a script like this. What do you think?

Reply:
I'm not sure how to do that, so maybe we can just merge this and you could tidy it up after?